docs/en/week12/12-3.md

```diff
@@ -0,0 +1,239 @@
+---
+lang-ref: ch.12-3
+title: MPC (EBM version)
+lecturer: Alfredo Canziani
+authors: Yang Zhou, Daniel Yao
+date: 28 Apr 2021
+---
+
+
+## Action plan
+- Model predictive control **[Here we are today]**
+    - Backprop through kinematic equation
+    - Minimisation of the latent
+- Truck backer-upper
+    - Learning an emulator of the kinematics from observations
+    - Training a policy
+- PPUU
+    - Stochastic environment
+    - Uncertainty minimisation
+    - Latent decoupling
+
+## State transition equations -- Evolution of the state
+
+Here we discuss a state transition equation where $x$ represents the state, $u$ represents control. We can formulate the state transition function in a continuous-time system where $x(t)$ is a function of continuous variable $t$.
+$\dot{x} = f(x,u)$
+$\frac{\partial x(t)}{\partial t} = f(x(t),u(t))$
+<center>
+<img src="{{site.baseurl}}/images/week12/12-3/figure1.png" style="background-color:#DCDCDC;" /><br>
+Figure 1: State and control illustration of a three-cycle
+</center>
+
+We use a tri-cycle as the example to study it. The orange wheel is the control $u$, $(x_c,y_c)$ is the instantaneous center of rotation. You can also have two wheels in the front. For simplicity, we use one wheel as the example.
+
+In this example $\boldsymbol{x}=(x,y,\theta,s)$ is the state, $\boldsymbol{u}=(\phi,\alpha)$ is the control.
+
+$$
+\left\{\begin{array}{l}
+\dot{x}=s \cos \theta \\
+\dot{y}=s \sin \theta \\
+\dot{\theta}=\frac{s}{L} \tan \phi \\
+\dot{s}=a
+\end{array}\right.e
+$$
+
+
+We can reformulate the differential equation from continuous-time system to discreate-time system
+
+$$
+\boldsymbol{x}[t]=\boldsymbol{x}[t-1]+f(\boldsymbol{x}[t-1], \boldsymbol{u}[t]) \mathrm{d} t
+$$
+
+To be clear, we show the units of $\boldsymbol{x}, \boldsymbol{u}$.
+$$
+\begin{array}{l}
+{[\boldsymbol{u}]=\left(\mathrm{rad}\  \frac{\mathrm{m}}{\mathrm{s}^{2}}\right)} \\
+{[\boldsymbol{x}]=\left(\mathrm{m} \  \mathrm{m} \  \mathrm{rad} \  \frac{\mathrm{m}}{\mathrm{s}}\right)}
+\end{array}
+$$
+
+Let's take a look at different examples. We use different color for variables we care about.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-3/figure2.png" style="background-color:#DCDCDC;" /><br>
+Figure 2: State Formulation
+</center>
+
+Example 1: Uniform Linear Motion: No acceleration, no steering
+<center>
+<img src="{{site.baseurl}}/images/week12/12-3/figure3.svg" style="background-color:#DCDCDC;" /><br>
+Figure 3: Control of Uniform Linear Motion
+</center>
+<center>
+<img src="{{site.baseurl}}/images/week12/12-3/figure4.svg" style="background-color:#DCDCDC;" /><br>
```

Comment:
The grid is a bit short. Please, extend to the end of the figure.
Here and below.

Reply:
Do you mean by horizontally?

Reply:
Both ways, see below.
![image](https://user-images.githubusercontent.com/2119355/117301059-3bdab780-ae48-11eb-9744-945bd02684e3.png)